testdata/serviceconfig.yml

```diff
@@ -21,7 +21,7 @@ rules:
           method: api.method | "unknown"
           response_code: response.http.code | "unknown"
       - descriptor_name:  request_latency
-        value: response.latency | "unknown"
+        value: response.latency | 0
```

Comment:
Hmm, yeah, and as far as I can tell Prometheus doesn't provide an easy way to filter out values of zero. That's unfortunate. Since request_latency is currently a counter (we really need to add histogram support), we still shouldn't use NaN if it sets the counter to NaN for forever: if the proxy doesn't report latency once we lose all latency data for forever.